### ISSUE TITLE
Dependency installation error

### ISSUE THREAD

**Issue by ghKZ** [arboleya/electrify#25]
When installing a dependency using the .electrify/package.json file

```
{
  "name": "my-electrified-app",
  "main": "index.js",
  "dependencies": {
    "electrify": "2.0.2",
    "serialport": "2.0.5"
  }
}
```

and then running `electrify`, it installs it but it builds a wrong version of it for Linux Mint 17 64bit.

It builds
`.electrify/node_modules/serialport/build/Release/node-v11-linux-x64`(note the v11)
but then once you `require('serialport')`, it tries to access
`.electrify/node_modules/serialport/build/Release/node-v46-linux-x64`
is of course isn't there so it throws and error and stop the app.

If I go into the `.electrify` folder manually and then issue `npm install serialport` it builds the correct(v46) version.

Since I'm kinda new to the node environment, is this something I have caused by doing something wrong or something else?

Node version
`nodejs -v`: `v4.2.2`
`node -v`: `v4.2.2`

Let me know if any additional information is required

**Comment by arboleya** [comment]
Hi there.

Makes sense, currently Electrify uses the NPM + NodeJS distributed with Meteor, to perform this kind of installs. This is like this because earlier versions of Electrify uses to run from inside Meteor, but now it has become a global NPM package as well, and we can start using the system `npm` and `nodejs` binaries for such tasks.

Tks for reporting, I'm gonna look into it.

**Comment by arboleya** [comment]
Hi, should be fixed in `2.1.2`, please try it out and let me know if everything went well.

**Comment by arboleya** [comment]
Closing, feel free to reopen in case something goes wrong.
